packages/kotti-ui/source/kotti-navbar/KtNavbar.vue

```diff
@@ -72,9 +72,11 @@ export default defineComponent<KottiNavbar.PropsInternal>({
 		NavbarNotification,
 		NavbarQuickLink,
 	},
+	// @ts-expect-error: official declarations of @types/vue-clickaway do
```

Comment:
I'm pretty sure I saw in your VSCode that this was detected as an unnecessary `@ts-expect-error`. It should therefore be a `@ts-ignore` since it's not always erroring